systers_portal/static/css/style.css

```diff
@@ -234,6 +234,22 @@ table.decoration-none  tr a {
   font-size: 1.3em;
 }
 
+/* Meetup Location CSS
+------------------------------------------------- */
+.box-boundary{
+  outline: 1px solid black;
+}
+
+.box-container{
+  margin: 10px 40px 10px;
+}
+
+.box-body{
+  font-size: 16px;
+  margin: 30px 0px;
+  text-align: justify;
```

Comment:
The **justify** align sometimes looks good, but since we have all sorts of browsers, most of the time browsers don't justify well. Though it works very well in professional typesetting software like Latex.